docs/docs/whatsnew.md

```diff
@@ -1,5 +1,19 @@
 # <b>Important updates</b>
 
+## v1.2.26
+
+### Support SBOM generation
+
```

Comment:
The Software Bill of Materials (SBOM) is a list of components, libraries, and module information that are required to build a software, and the supply chain relationships between them. An SBOM also lists the licenses that govern those components, versions of the components used in the codebase, and their patch status.